demo/src/screens/HomeScreen/HomeCarousel/HomeCarousel.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import {Carousel, Image} from 'react-bootstrap';
-import slideOne from '../../images/slide_one.png';
-import slideTwo from '../../images/slide_two.png';
-import slideThree from '../../images/slide_three.png';
+import {Link} from 'react-router-dom';
+import slideOne from '../../../images/slide_one.png';
+import slideTwo from '../../../images/slide_two.png';
+import slideThree from '../../../images/slide_three.png';
 import './HomeCarousel.css';
 
 /**
- * @return {!JSX} Slideshow component for showcasing
- *     our latest deals and offerings.
+ * Creates slideshow component for showcasing our latest deals and offerings.
+ * @return {!JSX} The component.
```

Comment:
Nit: Description "The component." is unnecessary.

> Method, parameter, and return descriptions (but not types) may be omitted if they are obvious from the rest of the method’s JSDoc or from its signature.

https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments